test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.test.js

```diff
@@ -1739,6 +1739,36 @@ describe('#compileMethods()', () => {
         .ApiGatewayMethodUsersCreatePost.Properties
     ).to.not.have.key('OperationName');
   });
+
+  it('should depends on permission resource', async () => {
```

Comment:
Let's move it to this `describe`:
```
describe('#compileMethods v2()', () => {
```

as it's indended to capture all new tests written with `runServerless` approach